src/modules/markets/MarketAssetsListItem.tsx

```diff
@@ -46,6 +47,7 @@ export const MarketAssetsListItem = ({ ...reserve }: ComputedReserveData) => {
         </Box>
         {reserve.symbol === 'AMPL' && <AMPLToolTip />}
         {reserve.symbol === 'renFIL' && <RenFILToolTip />}
+        {reserve.symbol === 'stETH' && <StETHCollateralToolTip />}
```

Comment:
I'd suggest removing it here since this only applies to astETH holders and not general markets

Reply:
Asked @iamanastasia about this, i don't have a strong opinion about this, i can remove it if we want.